### ISSUE TITLE
Can C3 generate 32 bit binaries and libraries for Windows?

### ISSUE THREAD

**Issue by heavymetalmixer** [c3lang/c3c#1613]
I saw in the repo README that C3 can compile to "Win32 x64". Does this mean that it can generate 32 and 64 bits binaries and libraries? Or only 64 bits ones?

![Captura de pantalla 2024-11-10 145735](https://github.com/user-attachments/assets/515aaa48-f7b8-4b8f-b555-9b2dee0b1ee4)

**Comment by lerno** [comment]
Only the x64 ones. Theoretically one could get the 32 bit going, but it's so hard to test it and there is such limited interest in it, that it seems like a waste spending time supporting it right now.

**Comment by heavymetalmixer** [comment]
> Only the x64 ones. Theoretically one could get the 32 bit going, but it's so hard to test it and there is such limited interest in it, that it seems like a waste spending time supporting it right now.

Limited interest? What about old C and C++ code made for 32 bits that could be ported to C3?

**Comment by lerno** [comment]
I am not sure I understand the scenario. Are you porting code to run on 32 bit computers?

**Comment by heavymetalmixer** [comment]
> I am not sure I understand the scenario. Are you porting code to run on 32 bit computers?

No, I mean code that was originally in 32 bits, and to compile it in 32 bits with C3 to not change its functionality (and not make it eat more resources).

**Comment by lerno** [comment]
This is certainly something that *eventually* could be added, but because of the extra work and testing that would need to go into this it's not a priority. As you might also see, various variants of 32 bit ARM isn't supported either for similar reasons.
